test/test_monitor.py

```diff
@@ -71,17 +80,112 @@ def test__find_modified_entities_fileview(self):
             # )
 
 
+class TestModifiedContainer:
+    """Test modifying containers"""
+
+    def setup_method(self):
+        self.syn = Mock()
+        self.days = 1
+        self.now = datetime.now().replace(tzinfo=tz.tzutc()).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        self.past = (datetime.now().replace(tzinfo=tz.tzutc()) - timedelta(days=self.days+1)).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        self.project = Project(name="test_project", parentId = "syn0", modifiedOn=self.now)
+        self.folder = Folder(name="test_folder", parentId = "syn1", modifiedOn=self.now)
+        self.file = File(name="test_file", parentId = "syn2", modifiedOn=self.now)
+
+
+    def test__traverse_folder_include(self):
+        """Traverse folder no children"""
+        with patch.object(self.syn, "get",
+                          return_value=self.folder) as patch_get,\
+            patch.object(self.syn, "getChildren",
+                         return_value=[]) as patch_child:
+            desc = monitor._traverse(self.syn, self.folder["parentId"])
+            patch_get.assert_called()
+            patch_child.assert_called()
+            assert desc == [self.folder["parentId"]]
+
+
+    def test__traverse_folder_exclude(self):
+        """Traverse folder no children"""
+        with patch.object(self.syn, "get",
+                          return_value=self.folder) as patch_get,\
+            patch.object(self.syn, "getChildren",
+                         return_value=[]) as patch_child:
+            desc = monitor._traverse(self.syn, self.folder["parentId"], include_types=["file"])
+            patch_get.assert_called()
+            patch_child.assert_called()
+            assert desc == []
+
+
+    def test__find_modified_entities_folder_modified(self):
+        """Find modified entities in a folder"""
+        with patch.object(self.syn, "get",
+                          return_value=self.folder) as patch_get,\
+            patch.object(self.syn, "getChildren",
```

Comment:
These should mock the _traverse function. Since you've tested _traverse already.

Reply:
Modified the _find_modified_entities_container() tests to patch both _traverse() and _find_modified_entities_file() since both these functions are tested already in other tests